docs/spire.md

```diff
@@ -0,0 +1,255 @@
+
+When SPIRE TaskRun result attestations is enabled, all TaskRuns will produce a signature alongside its results, which can then be used to validate its provenance. For example, a TaskRun result would be:
+```
+$ tkn tr describe cache-image-pipelinerun-8dq9c-fetch-from-git
+...
+<truncated>
+...
+📝 Results
+
+ NAME                    VALUE
+ ∙ RESULT_MANIFEST       commit,url,SVID,commit.sig,url.sig
+ ∙ RESULT_MANIFEST.sig   MEUCIQD55MMII9SEk/esQvwNLGC43y7efNGZ+7fsTdq+9vXYFAIgNoRW7cV9WKriZkcHETIaAKqfcZVJfsKbEmaDyohDSm4=
+ ∙ SVID                  -----BEGIN CERTIFICATE-----
+MIICGzCCAcGgAwIBAgIQH9VkLxKkYMidPIsofckRQTAKBggqhkjOPQQDAjAeMQsw
+CQYDVQQGEwJVUzEPMA0GA1UEChMGU1BJRkZFMB4XDTIyMDIxMTE2MzM1MFoXDTIy
+MDIxMTE3MzQwMFowHTELMAkGA1UEBhMCVVMxDjAMBgNVBAoTBVNQSVJFMFkwEwYH
+KoZIzj0CAQYIKoZIzj0DAQcDQgAEBRdg3LdxVAELeH+lq8wzdEJd4Gnt+m9G0Qhy
+NyWoPmFUaj9vPpvOyRgzxChYnW0xpcDWihJBkq/EbusPvQB8CKOB4TCB3jAOBgNV
+HQ8BAf8EBAMCA6gwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1Ud
+EwEB/wQCMAAwHQYDVR0OBBYEFID7ARM5+vwzvnLPMO7Icfnj7l7hMB8GA1UdIwQY
+MBaAFES3IzpGDqgV3QcQNgX8b/MBwyAtMF8GA1UdEQRYMFaGVHNwaWZmZTovL2V4
+YW1wbGUub3JnL25zL2RlZmF1bHQvdGFza3J1bi9jYWNoZS1pbWFnZS1waXBlbGlu
+ZXJ1bi04ZHE5Yy1mZXRjaC1mcm9tLWdpdDAKBggqhkjOPQQDAgNIADBFAiEAi+LR
+JkrZn93PZPslaFmcrQw3rVcEa4xKmPleSvQaBoACIF1QB+q1uwH6cNvWdbLK9g+W
+T9Np18bK0xc6p5SuTM2C
+-----END CERTIFICATE-----
+ ∙ commit       aa79de59c4bae24e32f15fda467d02ae9cd94b01
+ ∙ commit.sig   MEQCIEJHk+8B+mCFozp0F52TQ1AadlhEo1lZNOiOnb/ht71aAiBCE0otKB1R0BktlPvweFPldfZfjG0F+NUSc2gPzhErzg==
+ ∙ url          https://github.com/buildpacks/samples
+ ∙ url.sig      MEUCIF0Fuxr6lv1MmkreqDKcPH3m+eXp+gY++VcxWgGCx7T1AiEA9U/tROrKuCGfKApLq2A9EModbdoGXyQXFOpAa0aMpOg=
+```
+
+SPIRE TaskRun result attestations is currently an alpha experimental feature. 
+
+## Architecture Overview
+
+Since this feature relies on a SPIRE installation, we will show how it integrates into the architecture of Tekton.
+
+```
+┌─────────────┐  Register TaskRun Workload Identity           ┌──────────┐
+│             ├──────────────────────────────────────────────►│          │
+│  Tekton     │                                               │  SPIRE   │
+│  Controller │◄───────────┐                                  │  Server  │
+│             │            │ Listen on TaskRun                │          │
+└────────────┬┘            │                                  └──────────┘
+ ▲           │     ┌───────┴───────────────────────────────┐     ▲
+ │           │     │           Tekton TaskRun              │     │
+ │           │     └───────────────────────────────────────┘     │
+ │  Configure│                                          ▲        │ Attest
+ │  Pod &    │                                          │        │   +
+ │  check    │                                          │        │ Request
+ │  ready    │     ┌───────────┐                        │        │ SVIDs
```

Comment:
```suggestion
 │ Configure │                                          ▲        │ Attest
 │ Pod &     │                                          │        │   +
 │ check     │                                          │        │ Request
 │ ready     │     ┌───────────┐                        │        │ SVIDs
```

Previously 2 spaces on the left no spaces on the right.
Balanced to 1 space either side